src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -214,7 +214,7 @@ private function getBundleHierarchy(ContainerBuilder $container, array $config)
             }
             $container->addResource(new FileExistenceResource($dir));
 
-            if (null === $bundle['parent']) {
+            if (!isset($bundle['parent']) || null === $bundle['parent']) {
```

Comment:
Isn't this condition the same as `!isset($bundle['parent'])` ? Because if `null === $bundle['parent']`, then `!isset($bundle['parent'])` should be `true`.

Unless I'm reading it wrong...

Reply:
Good catch, `isset()` returns false for a value that's null.